src/publishers/bank-holidays-publisher.ts

```diff
@@ -0,0 +1,85 @@
+import moment from 'moment';
+
+import { TWhosOut } from '..';
+import {
+  FRIDAY_ISO_WEEKDAY,
+  HUMAN_READABLE_DATE,
+  MONDAY_ISO_WEEKDAY,
+} from '../common';
+import { postSlackMessage } from '../http';
+
+export const pusblishBankHolidays = async (
+  bankHolidays: TWhosOut,
+  today: moment.Moment
+): Promise<void> => {
+  const monthBankHolidaysBlocks: any[] = [];
+  let nextBankHolidaysBlocks: any[] = [];
+
+  if (
+    bankHolidays.month.length > 0 &&
+    ((today.date() === 1 && today.isoWeekday() <= FRIDAY_ISO_WEEKDAY) ||
+      (today.date() === 2 && today.isoWeekday() === MONDAY_ISO_WEEKDAY) ||
+      (today.date() === 3 && today.isoWeekday() === MONDAY_ISO_WEEKDAY))
+  ) {
+    monthBankHolidaysBlocks.push({
+      type: 'context',
+      elements: [
+        {
+          type: 'mrkdwn',
+          text: '*This month bank holidays:*',
+        },
+      ],
+    });
+
+    monthBankHolidaysBlocks.push(
+      ...bankHolidays.month.map(holiday => ({
+        type: 'section',
+        text: {
+          type: 'mrkdwn',
+          text: `*${holiday.name}*\n${moment(holiday.start).format(
+            HUMAN_READABLE_DATE
+          )}`,
+        },
+      }))
+    );
+
+    monthBankHolidaysBlocks.push({
+      type: 'divider',
+    });
```

Comment:
There's no need of this, please feel free to ignore it.
```suggestion
    const bankHolidaysHeader = {
      type: 'context',
      elements: [
        {
          type: 'mrkdwn',
          text: '*This month bank holidays:*',
        },
      ],
    };
    
    const holiday2Block = holiday => ({
        type: 'section',
        text: {
          type: 'mrkdwn',
          text: `*${holiday.name}*\n${moment(holiday.start).format(
            HUMAN_READABLE_DATE
          )}`,
        },
      });
      
    const bankHolidaysData = bankHolidays.map(holiday2Block);
    
    const dividerBlock = {
      type: 'divider',
    };

    monthBankHolidaysBlocks = [
      ...monthBankHolidaysBlocks,
      bankHolidaysHeader,
      ...bankHolidaysData,
      dividerBlock,
      ];
```